ambry-api/src/main/java/com/github/ambry/config/MySqlNamedBlobDbConfig.java

```diff
@@ -29,6 +29,33 @@ public class MySqlNamedBlobDbConfig {
   public static final String  QUERY_STALE_DATA_MAX_RESULTS = PREFIX + "query.stale.data.max.results";
   public static final String  STALE_DATA_RETENTION_DAYS = PREFIX + "stale.data.retention.days";
   public static final String TRANSACTION_ISOLATION_LEVEL = PREFIX + "transaction.isolation.level";
+  public static final String LIST_NAMED_BLOBS_SQL = "list.named.blobs.sql";
+
+  /**
+   * List named-blobs query.
+   * The inner query selects blobs based on the filters, and groups them on blob_name.
+   * In each group, it selects the blob with the highest version.
+   * The outer query filters out blobs that have been deleted, and return the latest blob in each group that is ready.
+   */
+  @Config(LIST_NAMED_BLOBS_SQL)
```

Comment:
What is hard to track?

Reply:
I think keeping all the query in the MySqlNamedBlobDb class is straight forward instead of keep it in the config file. we can discuss this internally.

Reply:
done